app/src/main/java/io/dolby/rtsviewer/ui/detailInput/DetailInputScreen.kt

```diff
@@ -195,15 +198,14 @@ fun DetailInputScreen(
 
                 Spacer(modifier = modifier.height(8.dp))
 
-                if (uiState.recentStreams.isNotEmpty()) {
-                    StyledButton(
-                        buttonText = stringResource(id = R.string.saved_streams_button),
-                        onClickAction = {
-                            onSavedStreamsClick()
-                        },
-                        buttonType = ButtonType.SECONDARY
-                    )
-                }
+                StyledButton(
```

Comment:
This button works, I wonder if splitting it into a TextView and a Button or a row of buttons would be a better UI

option a)
TextView : Current Env - ${env}
Button : Change Env
  onAction: cycle thru Env
  
  option b)
  Row of Buttons: Prod, Staging, Dev
    Change background color of currently selected Environment

Reply:
I will keep it as is for now, will get a better UI later